tailwind.config.ts

```diff
@@ -37,6 +37,19 @@ const tailwindConfig = {
       padding: {
         5.5: '22px'
       },
+      boxShadow: {
+        'custom-light': '0 2px 1px rgba(128, 128, 128, 0.472)',
+        'custom-dark': '0 4px 2px rgba(128, 128, 128, 0.472)',
+      },
+      maxHeight: {
+        '1000px': '1000px',
+      },
+      fontSize: {
```

Comment:
keep these variables local please, imagine everyone and every styling config dumped into the global tailwind config.